app/graphql/mutations/user_books/patch_user_book.rb

```diff
@@ -0,0 +1,29 @@
+module Mutations
+  module UserBooks
+    class PatchUserBook < Mutations::BaseMutation 
+      argument :user_id, Integer, required: true
+      argument :book_id, Integer, required: true 
+      argument :borrower_id, Integer, required: true
+      argument :status, Integer, required: true
+
+      field :success, String, null: false
+      
+      def resolve(user_id:, book_id:, borrower_id:, status:)
+        user_book = UserBook.find_by(user_id: user_id, book_id: book_id)
+
+        if user_book.nil?
+          {
+            success: 'False'
+          }
```

Comment:
Maybe we should add a bit more info here like errors: "Record not found" or something like that.

Reply:
Sweet, love the specific error message